docs/app/Components/ComponentDoc/ComponentDoc.js

```diff
@@ -68,14 +76,28 @@ class ComponentDoc extends Component {
 
   handleSidebarItemClick = (e, { path }) => {
     const { history } = this.props
-    const aPath = _.kebabCase(_.last(path.split('/')))
+    const aPath = _.kebabCase(path.split('/').join(' '))
```

Comment:
Let's pull this into a utility function in `docs/apps/utils`.  Something like `createComponentHash()`.  It would be ideal to be able to pass an example filename or file path to a single function that knows how to create the hash.  This way it can be used everywhere we are constructing the hash.

Also, we need to add redirects from the old hashes to the new hashes.  A utility function could be used for transforming the old hashes to the new hashes as well.